public/js/connectivity-manager.js

```diff
@@ -49,8 +43,8 @@ export default class Offline {
    * @param url the url to be checke for availability
    * @returns true if the user is online or the URL is cached
    */
-  static isAvailable(href) {
-    if (!href || window.navigator.onLine) return Promise.resolve(true);
+  isAvailable(href) {
+    if (!href || this.window.navigator.onLine) return Promise.resolve(true);
     return caches.match(href)
```

Comment:
I think it would be better to search the `toolbox.options.cache.name` cache only here, to prevent confusing bugs in future. `caches.match(.)` searches all caches, but the SW (and `sw-toolbox`) only searches the `toolbox.options.cache.name` cache. So if there's some synchronisation problem with the caches, `isAvailable()` could return true, even though the resource will not be returned by the SW.